lib/routes/jou/utils/index.ts

```diff
@@ -35,17 +35,28 @@ async function getItems(ctx, url, host, tableClass, timeStyleClass1, titleStyleC
             // 使用缓存
             cache.tryGet(item.link, async () => {
                 const response = await got({ url: item.link, https: { rejectUnauthorized: false } });
-                if (response.redirectUrls.length) {
+                if (response.redirectUrls && response.redirectUrls.length > 0) {
```

Comment:
`res.redirectUrls` will always be `undefined` since #14922. `'@/utils/got'` is now a partial-got-like wrapper of [ofetch](https://github.com/unjs/ofetch).